pocs/camera/camera.py

```diff
@@ -219,8 +222,51 @@ def take_observation(self, observation, headers=None, filename=None, *args, **kw
 
         return observation_event
 
-    def take_exposure(self, *args, **kwargs):
-        raise NotImplementedError
+    def take_exposure(self,
+                      seconds=1.0 * u.second,
+                      filename=None,
+                      dark=False,
+                      blocking=False,
+                      *args,
+                      **kwargs):
+        """
+        Take an exposure for given number of seconds and saves to provided filename.
+
+        Args:
+            seconds (u.second, optional): Length of exposure
+            filename (str, optional): Image is saved to this filename
+            dark (bool, optional): Exposure is a dark frame (don't open shutter), default False
```

Comment:
Not entirely no effect, I think.  After my  tinkering with the `take_exposure` methods I think it will, now, effect the value of the `IMAGETYP' keyword in the FITS header. Before the keyword was just absent on cameras that don't support taking dark frames internally but it occurred to me that you might still be taking a dark by other means (e.g. put the lens cap on your DSLR lens, selected a filter wheel position with an opaque dummy filter in it) in which case you definitely would want that keyword to appear, with the value 'Dark Frame', if `dark==True`.